pkg/trace/api/api.go

```diff
@@ -272,7 +269,7 @@ func (r *HTTPReceiver) listenTCP(addr string) (net.Listener, error) {
 		}()
 		return ln, err
 	}
-	return tcpln, err
+	return NewMeasuredListener(tcpln, "tcp_connections"), err
```

Comment:
We removed the `Close()` calls above. Does the MeasuredListener ever get Closed?

Reply:
Yes, I discovered that with a panic. When the HTTP server [shuts down](https://github.com/golang/go/blob/master/src/net/http/server.go#L2722), it automatically [closes all listeners](https://github.com/golang/go/blob/ab7c904bf2bb883b83001d8bb188d1809d74eee6/src/net/http/server.go#L2799).